src/test/java/org/session4/homework/PersonTest.java

```diff
@@ -11,13 +11,13 @@ class PersonTest {
     @DisplayName("This person is an adult")
     void testDisplayIfThePersonIsAdult(){
```

Comment:
There aren't naming conventions regarding test methods. Each team has it's own flavours, but is a good practice to have a construction like the one from below:
- `test` + `logic/method to test` + `shouldPass/shouldFail`

In our case, we can have `testAdulthoodOfPersonShouldPass`, because we are actually testing a specific method, and we can have either positive or negative tests.

Reply:
not sure if the addition of Pass/fail makes sense because we can use a better description with @DisplayName. Moreover the pass/fail is the output of the test, which is highlighted by the framework.